template.php

```diff
@@ -2,15 +2,13 @@
 
 /**
  * @file
- * Template overrides as well as (pre-)process and alter hooks for the
- * OCHA Basic theme.
+ * Template overrides, (pre-)process and alter hooks for the OCHA Basic theme.
  */
 
 /**
  * Implements hook_preprocess_page().
  */
-
-function ocha_basic_form_alter(&$form, &$form_state, $form_id) {
+function ochabasic_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
I'm seeing that all the hooks are renamed, but the `ocha_basic.info` file isn't being renamed. I'm not 100% sure, but I think this will result in the hooks not being applied (unless the `ochabasic` module that @cafuego installed enables them to be picked up. but it will go away if I correctly understood the convo about the dev site earlier)